docker/setup-test-wp.sh

```
@@ -0,0 +1,105 @@
+WP_TECH2_PASS="${WP_TECH2_PASS:-techpass123}"
+WP_TECH2_EMAIL="${WP_TECH2_EMAIL:-tech2@example.com}"
+CLIENT1_NAME="${CLIENT1_NAME:-Test Client One}"
+CLIENT1_EMAIL="${CLIENT1_EMAIL:-client1@example.com}"
+CLIENT2_NAME="${CLIENT2_NAME:-Test Client Two}"
+CLIENT2_EMAIL="${CLIENT2_EMAIL:-client2@example.com}"
+
+DC="docker compose -f docker-compose.test.yml"
+WP="$DC exec -T wpcli wp --allow-root --path=/var/www/html"
+
+echo "→ Waiting for WordPress to be ready..."
+for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)